src/Engine/Army.java

```diff
@@ -253,9 +252,9 @@ public Color getMarkingColor(Unit unit)
     return Color.RED;
   }
 
-  protected boolean awbwDeniesCharge()
+  protected boolean taggingDeniesCharge()
   {
-    return TagMode.AWBW == gameRules.tagMode
+    return gameRules.tagMode.supportsMultiCmdrSelect
         && getAbilityText().length() > 0;
   }
```

Comment:
I doublechecked AWDS, and realized I was mistaken about the secondary CO getting charge being gained during the primary CO's COP.
This does have the weird effect that COs who can normally charge during a power can't if tagging is on, but I dunno how I want it to work.

Reply:
I assume this discrepancy (COs who can/can't charge during a power) only exists across games, and not within them?

Reply:
There are literally no COs in a cart game that can charge during a power.
It's a common suggested D2D for Adder/custom CO mechanic, but hasn't been in an official game.

Reply:
Wait, then why mention "COs who can normally charge during a power" when there is no such thing?

Reply:
It's currently a thing (for our custom COs), though that could be changed.
But also I don't want to rule it out as a mechanic.

Reply:
That's ... pretty niche, but _okay_